setuper/default_healthFacility_form.py

```diff
@@ -0,0 +1,126 @@
+import uuid
+from datetime import datetime
+from submissions import submission2xml
+from fake import fake_person
+import random
+
+
+def setup_health_facility_level_default_form(account_name, iaso_client):
+    print("-- Setting up a default form for Health Facility level")
+    project_id = iaso_client.get("/api/projects/")["projects"][0]["id"]
+    org_unit_types = iaso_client.get("/api/v2/orgunittypes/")["orgUnitTypes"]
+    hf_out = [out for out in org_unit_types if out["name"] == "Formation sanitaire"][0]
```

Comment:
```suggestion
    health_facility_type = [out for out in org_unit_types if out["name"] == "Formation sanitaire"][0]
```

This is easier to understand when you're reading the code for the first time.